src/coprocessor/dag/expr/builtin_math.rs

```diff
@@ -251,6 +258,45 @@ mod test {
         }
     }
 
+    #[test]
+    fn test_round() {
+        let tests = vec![
+            (
+                ScalarFuncSig::RouldReal, 
+                Datum::F64(3.45), 
+                Datum::F64(3f64)
+            ),
+            (
+                ScalarFuncSig::RouldReal,
+                Datum::F64(-3.45),
+                Datum::F64(-3f64),
+            ),
+            (
+                ScalarFuncSig::RouldReal,
+                Datum::F64(f64::MAX),
+                Datum::F64(f64::MAX),
+            ),
+            (
+                ScalarFuncSig::RouldReal,
```

Comment:
Could you help update the [tipb](https://github.com/pingcap/tipb ) to the newest version to rename RouldRead to RoundReal?

Reply:
ok

Reply:
i found that these is an error showing "no method named `set_encode_type` found for type `tipb::select::StreamResponse` " when compiling tikv after tipb updated.

Reply:
got it, I'll update it later then.

Reply:
When u have updated,tell me then.